apps/docs/docs/message-db/source.md

```diff
@@ -0,0 +1,61 @@
+# Source
+
+The MessageDB source continuously fetches batches of messages and dispatches
+them to a supplied handler and dispatches them to a supplied handler. It
+supports reading from multiple categories at a time, server side filtering, and
+splitting processing into multiple parallel consumer groups.
+
+## Options
+
+When creating a MessageDB source via `MessageDbSource.create` the following
+options can be supplied
+
+```ts
+interface CreateOptions {
+  /** The database pool to use to read messages from the category */
+  pool: Pool
+  /** The categories to read from */
+  categories: string[]
+  /** The maximum number of messages to read from the category at a time
+   * @default 500
+   */
+  batchSize?: number
+  /** The name of the consumer group to use for checkpointing */
+  groupName: string
+  /** The checkpointer to use for checkpointing */
+  checkpointer: ICheckpointer
+  /** The handler to call for each batch of stream messages */
+  handler: (streamName: string, events: ITimelineEvent<string>[]) => Promise<void>
+  /** The number of milliseconds to sleep between tail reads */
```

Comment:
more ai at work!

sleep time in ms, when reading has reached the end of the category's events